UDV-Core/src/Modules/Links/Model/Link.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Represents a link between a document and a target (in this example, the
+ * only valid target types are city objects).
+ */
+export class Link {
+  /**
+   * Constructs an empty link.
+   */
+  constructor() {
+    /**
+     * The ID of the link.
+     * 
+     * @type {number}
+     */
+    this.id;
+
+    /**
+     * The source (document) ID.
+     * 
+     * @type {number}
+     */
+    this.source_id;
+
+    /**
+     * The target ID. For the moment, the only targets are city objects.
+     * 
+     * @type {number}
+     */
+    this.target_id;
+
+    /**
+     * The X coordinate of the centroid.
+     * 
+     * @type {number}
+     */
+    this.centroid_x;
+
+    /**
+     * The Y coordinate of the centroid.
+     * 
+     * @type {number}
+     */
+    this.centroid_y;
+
+    /**
+     * The Z coordinate of the centroid.
+     * 
+     * @type {number}
+     */
+    this.centroid_z;
+  }
```

Comment:
Is this centroïd needed to travel to the cityobject ? If yes, could you add a comment about that please ?

Reply:
Yes, I updated the comments to explain